assets/src/common/helpers/index.js

```diff
@@ -336,22 +297,6 @@ export const mediaLibraryHasTwoNotices = function() {
 	return Boolean( this.secondary.get( FILE_TYPE_ERROR_VIEW ) ) && Boolean( this.secondary.get( FILE_SIZE_ERROR_VIEW ) );
 };
 
-/**
- * Gets the number of megabytes per second for the video.
- *
- * @param {Object} media The media object of the video.
- * @return {?number} Number of megabytes per second, or null if media details unavailable.
- */
-export const getVideoBytesPerSecond = ( media ) => {
```

Comment:
Also need to remove `enforceFileSize` function above.

Attention, `enforceFileSize` is used in `EnforcedFileToolbarSelect`, which needs to be updated accordingly.